lib/pronto/formatter/text_formatter.rb

```diff
@@ -14,9 +30,30 @@ def location(message)
         line = message.line
         lineno = line.new_lineno if line
         path = message.path
-        commit_sha = message.commit_sha[0..6] if message.commit_sha
+        commit_sha = message.commit_sha
+
+        if path || lineno
+          format_location_path(path, lineno)
+        elsif commit_sha
+          format_location_commit(commit_sha)
+        else
+          raise 'Invalid message location'
```

Comment:
> No exceptions were thrown previously, it would have just returned nil for location if the commit_sha wasn't present.

Correct! I've rolled back to this behavior (with updated spec).